src/control/SConscript

```diff
@@ -57,6 +56,18 @@ def get_build_flags(benv):
         flags += "-race"
     return flags
 
+def install_go_manpage(menv, name):
+    """
+    Dynamically generate a manpage for a tool.
+    """
+    gen_bin = join('$BUILD_DIR', 'src/control/bin', name)
+    build_path = join('$BUILD_DIR', 'src/control', name+'.8')
+    install_path = join(menv.subst("$PREFIX"), "share/man/man8", name+'.8')
+    menv.Command(build_path, [gen_bin, gurt_lib],
```

Comment:
why do you need gurt_lib here?

Reply:
I think because otherwise I was running into trouble with it not being built first? That's what I remember, but my approach to working with scons is to bang some rocks together until I get the result I'm hoping for, rather than having a deep understanding of what's happening.